docs/topics/ref_breakdown-of-package-json-properties.adoc

```diff
@@ -1,5 +1,5 @@
 [id='breakdown-of-package-json-properties_{context}']
-= Breakdown of Package.json Properties
+= Breakdown of Package.json properties
```

Comment:
Package.json is not a name, and actually the file is called "package.json". Should be lower-case here.

Reply:
Fixed.